component/pdp/capabilities.go

```diff
@@ -80,15 +87,11 @@ func evalInteraction(
 		fhir.TypeRestfulInteractionSearchType,
 	}
 	if !slices.Contains(supported, input.InteractionType) {
```

Comment:
I just noticed that the interaction type is passed by the PEP to the PDP, but I don't think that's correct (at least not in most cases); as that requires the PEP to have in-depth detail about FHIR and do some mapping/parsing (e.g. GET without search params becomes "read", with search params it becomes a "search"). That makes implementing a PEP very complicated.

I think that logic should be in the PDP, making the PEP->PDP request body just user info + HTTP request info (basically URL and query parameters, top of mind), just like when we'd employ OPA + NGINX (for instance).

We might have this FHIR input as intermediate data model for the FHIR policies, but I don't think it should be the API input of the PDP as sent by the PEP.